python/pyspark/sql/connect/dataframe.py

```diff
@@ -951,6 +951,39 @@ def createOrReplaceGlobalTempView(self, name: str) -> None:
         ).command(session=self._session)
         self._session.execute_command(command)
 
+    def rdd(self, *args: Any, **kwargs: Any) -> None:
+        raise NotImplementedError("RDD Support for Spark Connect is not implemented.")
```

Comment:
Maybe keep consistently to include `for Spark Connect` or not include in the error message? 

Only this one mentions `Spark Connect` now.

Reply:
While we might implement the others, here I want to be really specific about not being part of Spark Connect.

Reply:
Sounds good